### ISSUE TITLE
DesignWare ARC EM Starter kit

### ISSUE THREAD

**Issue by ranahack** [foss-for-synopsys-dwc-arc-processors/toolchain#30]
Hello,

I am a student a the University of Chile, a part-time Profesor from my university works at Synopsys Chile and he lent me a EM Starter kit, to get familiar with it. But I have not found too much information about it, also the getting started manual that comes with the kit doesnt help much. My Profesor also has been having problems on getting the MetaWare program and license. So that why I am trying to use the GNU toolchain, but actually I dont know how to setup a program for this platform and compile it. So if you have a tutorial video or pdf? I would really appreciate if you could send it to me, so I can get started with the kit. 

Best regards
René Espinoza
Electrical Engineer student 
Universidad de Chile
Email: rene.espinozaj@gmail.com

**Comment by anthony-kolesov** [comment]
Hi René,

Yes, documentation on how to use GNU Toolchain and EM Starter Kit is lacking, this is something that is going to be addressed in next releases of both products.

Early next week we are going to release first Release Candidate of GNU Toolchain 4.8-R2, with final release in next three weeks. It will be provided as a prebuilt package for both Windows and Linux, which will save you from building toolchain yourself. We are currently working on documentation and I will update this issue with links when it will be ready.

In a nutshell you need to have GNU tools (REAME.md in this repository describes the process of building toolchain) and [ARC port of OpenOCD](https://github.com/foss-for-synopsys-dwc-arc-processors/openocd), file doc/README.ARC there describes building and usage process. But I recommend to wait until prebuilt toolchain and final documentation will be available.

We'll try to find out what is necessary to provide you Metaware license as well.

**Comment by ranahack** [comment]
Thanks a lot Anthony. 

About the Metaware license, do you need the contact of my guide teacher that works at Synopsys?

**Comment by anthony-kolesov** [comment]
I still don't have any response from the manager who knows about the subject of Metaware licences for Chile. Probably contact of your teacher will be needed. You can send contacts to Anton.Kolesov (at) synopsys.com if they should be kept away from public discussion.

**Comment by anthony-kolesov** [comment]
Got a reply from Marketing. No need to send me anything, your problem will be resolved.

**Comment by ranahack** [comment]
Thank you!

**Comment by ranahack** [comment]
Hello Anthony,

I have some questions you could help me with. Is the GNU Toolchain ready to use?  Also, is there any news with your marketing department about the metaware license? 

Best regards
René

**Comment by fbedard** [comment]
Hi Rene,

We asked for an update on your MetaWare licenses from our marketing department today and got a response that the SNPS University Program has donated several copies of the MetaWare Development Toolkit to your university on Nov 5th. 

Regards,

Francois

**Comment by ranahack** [comment]
Thank you Francois,

I will appreciate if you could you give me the name or the email of the one in charge that received the copies on my university?

regards,

René

**Comment by fbedard** [comment]
Hi Renee,

Contact person is Marcos Diaz (mdiazq@ing.uchile.cl) 

Francois

**Comment by anthony-kolesov** [comment]
Hello Rene,

Did you get a license? Can we close this issue?
